pennylane/transforms/tape_expand.py

```diff
@@ -25,6 +27,11 @@
     not_tape,
 )
 
+# Needed for custom decomposition context manager
+from pennylane.transforms.qfunc_transforms import NonQueuingTape
```

Comment:
I only learnt about this recently :)

Reply:
The `NonQueuingTape` is something we should definitely move out of the transforms folder! we should put it in `tape/` lol